src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,13 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+import './UserInfo.scss';
+
+export const UserInfo = ({ name, email }) => (
+  <>
+    <p>
+      {' Posted by  '}
+
+      <a className="UserInfo" href={email}>
```

Comment:
You should not use 'email' as the href for the anchor tag. The href attribute requires a URL. A 'mailto:' could be prepended to the email for the link to function correctly.